docs/content/releases.md

```diff
@@ -7,6 +7,36 @@ toc: true
 docs: "DOCS-616"
 ---
 
+## NGINX Ingress Controller 2.2.0
+
+12 Apr 2022
+
+FEATURES:
```

Comment:
deprecation of 'preview' should be repeated.

Reply:
it's in the `CHANGES`, that's where we usually have changes to the behavior of the IC. Is that enugh?